cdc/log.cc

```diff
@@ -1320,15 +1330,17 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
                     tracing::trace(tr_state, "CDC: Splitting {}", m.decorated_key());
                     generated_count = 0;
                     for_each_change(m, s, [&] (mutation mm, api::timestamp_type ts, bytes tuuid, int& batch_no) {
-                        mutations.push_back(trans.transform(std::move(mm), rs.get(), ts, tuuid, batch_no));
+                        auto [mut, parts] = trans.transform(std::move(mm), rs.get(), ts, tuuid, batch_no);
```

Comment:
Isn't compiler complain about unused variable parts?

Reply:
Nope, I tried compiling it and it does not complain.